src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -0,0 +1,208 @@
+     *
+     * @for ScopeModel
+     * @method amendAltitude
+     * @param radarTargetModel {RadarTargetModel}
+     * @param altitude {string}
+     * @return result {array} [success of operation, system's response]
+     */
+    amendAltitude(radarTargetModel, altitude) {
+        altitude = parseInt(altitude, DECIMAL_RADIX);
+
+        return radarTargetModel.amendAltitude(altitude);
+    }
+
+    /**
+     * Enable handlers
+     *
+     * @for ScopeModel
+     * @method disable
+     */
+    disable() {
+        this._eventBus.off(EVENT.SET_THEME, this._setTheme);
+    }
+
+    /**
+     * Disable handlers
+     *
+     * @for ScopeModel
+     * @method enable
+     */
+    enable() {
+        this._eventBus.on(EVENT.SET_THEME, this._setTheme);
+    }
+
+    /**
+     * Initiate a handoff to another sector
+     *
+     * @for ScopeModel
+     * @method initiateHandoff
+     * @param radarTargetModel {RadarTargetModel}
+     * @param sectorCode {string} the handoff code for the receiving sector
+     * @return result {array} [success of operation, system's response]
+     */
+    initiateHandoff(radarTargetModel, sectorCode) {
+        return [false, 'initiateHandoff command not yet available'];
+    }
+
+    /**
+     * Change direction and/or length of data block leader line
+     *
+     * @for ScopeModel
+     * @method moveDataBlock
+     * @param radarTargetModel {RadarTargetModel}
+     * @param commandArguments {array}
+     * @return result {array} [success of operation, system's response]
+     */
+    moveDataBlock(radarTargetModel, commandArguments) {
+        return radarTargetModel.moveDataBlock(commandArguments);
+    }
+
+    /**
+     * Toggle visibility of the data block of a given `RadarTargetModel`, on this
+     * sector's scope, or the scope of another sector
+     *
+     * @for ScopeModel
+     * @method propogateDataBlock
+     * @param radarTargetModel {RadarTargetModel}
+     * @param sectorCode {array} handoff code for the receiving sector
+     * @return result {array} [success of operation, system's response]
+     */
+    propogateDataBlock(radarTargetModel, sectorCode) {
+        return [false, 'propogateDataBlock command not yet available'];
+    }
+
+    /**
+     * Amend the route stored in the scope for a given `RadarTargetModel`
+     *
+     * @for ScopeModel
+     * @method route
+     * @param radarTargetModel {RadarTargetModel}
+     * @param routeString {string}
+     * @return result {array} [success of operation, system's response]
+     */
+    route(radarTargetModel, routeString) {
+        return [false, 'route command not yet available'];
+    }
+
+    /**
+     * Execute a scope command from a `ScopeCommandModel`
+     * @method runScopeCommand
+     * @param scopeCommandModel {ScopeCommandModel}
+     * @return result {array} [success of operation, system's response]
+     */
+    runScopeCommand(scopeCommandModel) {
+        const functionName = scopeCommandModel.commandFunction;
```

Comment:
I'm not a fan of using the words `function` and `arguments` as part of var names

Reply:
seems appropriate for the particular case, but I know what you mean. Any ideas?

Reply:
I suppose this is a very minor gripe and I can't really think of anything better at the moment.  

In the interest of moving this along, I'll allow it 😉